packages/@office-iss/react-native-win32/src-win/Libraries/Inspector/Inspector.win32.js

```diff
@@ -114,19 +110,15 @@ function Inspector({
         }
       }
 
-      // [Win32 Avoid Dimensions call
```

Comment:
Should this win32 difference be re-applied?

Reply:
Yes.  Dimensions.get('window') throws in win32 -- since Dimensions.get('window') is a bad API for multi window applications.